### PR TITLE
Convert most gke jobs to bootstrap

### PR DESCRIPTION
**Pull request by krzyzacy**
ref #1021 #1022 

will do blocking jobs next (#1058)

and some of them need to be clarified, I'll make a second pass.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1111)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml [hunk 1]
```diff
@@ -843,44 +352,9 @@
...
-                # TODO: Enable this when we've split 1.2 tests into another project.
-                export FAIL_ON_GCP_RESOURCE_LEAK="false"
-                export KUBE_GKE_IMAGE_TYPE="gci"
-    jobs:
-        - 'kubernetes-e2e-{gke-suffix}'
-
 # Jobs that run e2e tests on GKE with GCI as the node image on the release-1.2
 # branch.
-- project:
+- project:W
```
Comment (fejta) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml]: typo

**Comment by fejta** [comment]
/lgtm
otherwise

**Comment by krzyzacy** [comment]
> I'm assuming you updated timeouts correctly?

I hope so... although I might be dizzy to make mistakes... will double check
still fixing presubmits

**Comment by krzyzacy** [comment]
sudo /lgtm

**Comment by krzyzacy** [comment]
introducing more chaos!

**Pull request merged**